cfme/configure/configuration/diagnostics_settings.py

```diff
@@ -328,6 +334,16 @@ def collect_current(self):
 
 
 # ======================= Server Collect Logs ============================
+# common part of is_displayed check for ServerCollectLogsSlaveView and ServerCollectLogsMasterView
+def server_collect_logs_displayed(view, server, slave=True):
```

Comment:
okay, this is closer to what I thought about but I'd implement it a little bit different.

```python
class ServerCollectLogsView(ServerDiagnosticsView):
  title_template = 'Diagnostics Server "{name} {sid}"{current}'

  def form_expected_title(self, one, two, three):
    return self.title_template.format()

  @property
  def is_displayed(self):
    return in_server_collect_logs and self.title.text == self.form_expected_title()


class ServerCollectLogsSlaveView(ServerCollectLogsView):
  @property
    def is_displayed(self):
        return in_server_collect_logs and self.title.text == self.form_expected_title()  

class ServerCollectLogsMasterView(ServerCollectLogsView):
    @property
    def is_displayed(self):
        return in_server_collect_logs and self.title.text == self.form_expected_title()  
```

Reply:
@izapolsk  yep, yours is much better :+1: 
I'm gonna implement this

Reply:
yeah, it is a pity that we managed to wrap up only one statement. I thought it could be made more neat :(